src/utils/remoteImages.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ImageType,
+  RemoteImageInfo,
+  RemoteImageResult,
+} from '@jellyfin/sdk/lib/generated-client/models';
+import { Track } from '../types/track';
+
+export const chooseSuitableRemoteImage = (
+  remoteImageResult: RemoteImageResult,
+) => {
+  const primaryImages: RemoteImageInfo[] | undefined =
+    remoteImageResult.Images.filter((x) => x.Type === ImageType.Primary);
+
+  if (primaryImages.length > 0) {
+    return primaryImages[0];
+  }
+
+  if (remoteImageResult.Images.length > 0) {
+    return remoteImageResult.Images[0];
+  }
+
+  return undefined;
```

Comment:
the undefined here is not required (return; results by default in undefined as return statement)